__tests__/plugin-test.js

```diff
@@ -115,15 +115,20 @@ describe('release-it-yarn-workspaces', () => {
   let ROOT = process.cwd();
   let dir;
 
-  function setupProject(workspaces) {
+  function setupProject(workspaces, pkg = {}) {
     dir.write({
-      'package.json': json({
-        name: 'root',
-        version: '0.0.0',
-        license: 'MIT',
-        private: true,
-        workspaces,
-      }),
+      'package.json': json(
+        Object.assign(
+          {
+            name: 'root',
+            version: '0.0.0',
+            license: 'MIT',
+            private: true,
+            workspaces,
+          },
+          pkg
+        )
```

Comment:
Can we use object spread instead?

```suggestion
          {
            name: 'root',
            version: '0.0.0',
            license: 'MIT',
            private: true,
            workspaces,
            ...pkg
          },
```

Reply:
You'll have to bump the eslint `ecmaVersion` to `2018` in order to support the spread operator.

Reply:
linting was also failing, this i could fix by setting `ecmaVersion` to 2020 (which i think should be fine for node >= 14 in general).